components/Starknet/modules/ROOT/pages/index.adoc

```diff
@@ -1,6 +1,6 @@
 [id="overview"]
 = Introduction
 
-Starknet is a permissionless decentralized Validity-Rollup (also known as a “ZK-Rollup”). It operates as an L2 network over Ethereum, enabling any dApp to achieve unlimited scale for its computation – without compromising Ethereum’s composability and security, thanks to Starknet’s reliance on the safest and most scalable cryptographic proof system – STARK.
+Starknet is a permissionless decentralized Validity-Rollup (also known as a _ZK-Rollup_). It operates as an L2 network over Ethereum, enabling any dApp to achieve unlimited scale for its computation – without compromising Ethereum’s composability and security – thanks to Starknet’s reliance on the safest and most scalable cryptographic proof system: STARK.
```

Comment:
* Get rid of parentheses.
* I'm very cautious about using superlatives like _unlimited scale_ and _the safest and most scalable cryptographic proof system_. Check with legal.* 
* I wanted to see the source for the cards. Actually, I see you made some changes here that are not reflected in the preview...is there a disconnect?